crates/bevy_pbr/src/atmosphere/mod.rs

```diff
@@ -435,3 +444,22 @@ fn configure_camera_depth_usages(
         camera.depth_texture_usages.0 |= TextureUsages::TEXTURE_BINDING.bits();
     }
 }
+
+/// Selects how the atmosphere is rendered. Choose based on performance needs,
+/// scene scale, and how accurate you need shadows and long-distance lighting to be.
+#[derive(Clone, Default, Reflect, Copy)]
+pub enum AtmosphereRenderingMethod {
```

Comment:
How about `AtmosphereMode` `LookupTable` `Raymarched`

Reply:
Sounds good to me. Nice and simple, clean naming.

Reply:
Ended up using `LookupTexture` (not `LookupTable`)